doctoshotgun.py

```diff
@@ -196,6 +197,10 @@ def get_patients(self):
         self.master_patient.go()
 
         return self.page.get_patients()
+    
+    def remove_accents(self, string):
+        nfkd = unicodedata.normalize('NFKD', string)
+        return u"".join([c for c in nfkd if not unicodedata.combining(c)])
```

Comment:
Works well from what I've tested.

Two improvements that could be done as a followup would be to also:
* lowercase the string;
* replace non letters characters (maybe `/\W/`) with a dash (or at least the most common ones: spaces (` `) and apostrophe `'`).

examples:

```py
In [8]: remove_accents("Saint-Médard-en-Jalles")
Out[8]: 'Saint-Medard-en-Jalles'
# should be 'saint-medard-en-jalles' to work

In [9]: remove_accents("Arès")
Out[9]: 'Ares'
# should be 'ares' to work

In [10]: remove_accents("L'Haÿ les roses")
Out[10]: "L'Hay les roses"
# should be 'l-hay-les-roses' to work
```